src/schemas/SettingsSchema.js

```diff
@@ -13,6 +13,7 @@ module.exports = graphql.buildSchema(`
   type Mutation {
     createSite(input: SiteDefinition!): Site
     removeSite(input: SiteDefinition!): Site
+    createStream(input: StreamDefinition!): Stream
```

Comment:
I had to include a separate `Stream` type. If I returned a `StreamDefinition`, the `StreamDefinition` is an `input` type and the `params` fields holds an input type `ParamsEntry`, which results in an expected output type error: 
```
Error: Expected Output type.
    at invariant (C:\Users\stmarker\Documents\project-fortis\project-fortis-services\node_modules\graphql\jsutils\invariant.js:19
:11)
```